composer.json

```diff
@@ -139,20 +139,20 @@
             "@fix-cs",
             "@phpstan"
         ],
-        "check-cs": "packages/EasyCodingStandard/bin/ecs check packages --ansi",
+        "check-cs": "ecs check packages --ansi",
         "fix-cs": [
-            "packages/EasyCodingStandard/bin/ecs check packages --fix --ansi",
+            "ecs check packages --fix --ansi",
             "bin/clear_readmes.sh"
         ],
-        "phpstan": "vendor/bin/phpstan analyse packages --error-format symplify --ansi",
+        "phpstan": "phpstan analyse packages --error-format symplify --ansi",
         "changelog": [
-            "packages/ChangelogLinker/bin/changelog-linker dump-merges --in-categories --in-packages",
-            "packages/ChangelogLinker/bin/changelog-linker link",
-            "packages/ChangelogLinker/bin/changelog-linker cleanup"
+            "changelog-linker dump-merges --in-categories --in-packages",
+            "changelog-linker link",
+            "changelog-linker cleanup"
         ],
-        "merge": "packages/MonorepoBuilder/bin/monorepo-builder merge",
-        "propagate": "packages/MonorepoBuilder/bin/monorepo-builder propagate",
```

Comment:
I'm curious, will this still work? 
It's not in `vendor/bin/...` path so I don't think it will be found

Reply:
You're right, it's broken. I reverted. It will still be broken on windows but it's not required to automate testing